db/models.py

```diff
@@ -22,6 +29,11 @@ class Movie(models.Model):
     actors = models.ManyToManyField(to=Actor, related_name="movies")
```

Comment:
The `related_name` attribute in the `ManyToManyField` for `actors` is set to 'movies'. Ensure that this does not conflict with any other related names in your models. It's a good practice to use unique related names to avoid potential clashes.